README.md

```diff
@@ -255,6 +255,33 @@ Workbench schema lives in `api/db` --> all workbench related activities access/p
 
 CDR schema lives in `api/db-cdr` --> all cdr/cohort builder related activities access/persist data here
 
+## Generate cdr and public count databases for a CDR version
+
+This happens anytime a new cdr is released or if you want all the count data for databrowser and cohort builder generated locally. 
+
+### Generate count data in BigQuery from a cdr in biquery 
+`./project.rb generate-cdr-counts --bq-project all-of-us-ehr-dev \
+  --bq-dataset test_merge_dec26 --workbench-project all-of-us-workbench-test --public-project all-of-us-workbench-test --cdr-version 20180130 --bucket all-of-us-workbench-cloudsql-create`
```

Comment:
You have to create the bucket before you do this, right? Might want to mention that above.

Reply:
Noted.